packages/venia-concept/template.html

```diff
@@ -126,6 +126,7 @@
               unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
             }
         </style>
+        <script>window.adobeDataLayer = window.adobeDataLayer || []</script>
```

Comment:
Would this line be better in `venia-concept-data-layer/index.js` to avoid adding this array unless the extension is used? Like
```js
useEffect(() => {
        window.adobeDataLayer = window.adobeDataLayer || [];
        import(/* webpackChunkName: "acdl" */
        /* webpackMode: "lazy" */
        /* webpackPrefetch: true */
        '@adobe/adobe-client-data-layer');
    }, []);
```

Reply:
That might work, but I'm not really familiar with how things are going to use this data layer object, so for now I put the property definition as early as possible—prior to the React render, even. The only really important thing is that the library itself gets deferred as long as possible and the browser is completely idle.

Reply:
> [The BuildBus] runs the intercept files in dependency order. If your module declares another module with Targets in peerDependencies, the other module will intercept first.

If another extension relies on this object to be present, it should list `adobe-client-data-layer` as a `peerDependency` to ensure the `adobe-client-data-layer` extension intercepts first (and this array is added).

Reply:
@supernova-at @deloreyj Good points. Done.